cmd/gossamer/configcmd.go

```diff
@@ -71,6 +70,16 @@ func makeNode(ctx *cli.Context) (*dot.Dot, *cfg.Config, error) {
 		return nil, nil, fmt.Errorf("error loading state and runtime: %s", err)
 	}
 
+	// load extra genesis data from DB
+	gendata, err := state.Db().LoadGenesisData()
+	if err != nil {
+		return nil, nil, err
+	}
+
+	updateP2pConfig(&fig.P2p, gendata)
+
+	log.Info("🕸\t Starting gossamer...", "datadir", fig.Global.DataDir, "protocolID", fig.P2p.ProtocolId, "bootnodes", fig.P2p.BootstrapNodes)
```

Comment:
This doesn't really make sense here:
```
INFO[11-04|10:30:26] 🕸  Starting gossamer...               datadir=/home/d/.gossamer protocolID=/gossamer/0 bootnodes=[/ip4/104.211.54.233/tcp/30363/p2p/16Uiu2HAmFWPUx45xYYeCpAryQbvU3dY8PWGdMwS2tLm1dB1CsmCj]
INFO[11-04|10:30:26] 🕸️         Starting node...                name=gossamer
```
We should avoid logging this here as the node hasn't actually been started. IMO we should add something like this to `Dot.Start()` and remove the log in `main.go`. Lmk what you think.

Reply:
okay, could work, I would just have to add the fields to dot then. I don't really think it makes too much of a difference

Reply:
Maybe it could just be something more like `Configured node`, so it makes sense in `makeConfig`. Alternatively, maybe these should be logged in their respective services on startup

Reply:
changed